### PR TITLE
[DOC] Fix links to non-existent pages

### PR DESCRIPTION
**Pull request by BurdetteLamar**
Fixes the links I've found that link to non-existent pages (but not those with incorrect fragments):

- When I could find a useful substitute page, I've changed the link.
- When not, I've removed the link.
- In one file, the links were (I think) unintended;  I've moved that text  (code, actually) into code blocks.

### CODE REVIEW

File: README.ja.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ UNIXであれば `configure` がほとんどの差異を吸収してくれるは
 
 ## 配布条件
 
-[COPYING.ja](COPYING.ja) ファイルを参照してください．
+[COPYING.ja](https://docs.ruby-lang.org/en/master/COPYING_ja.html   ) ファイルを参照してください．
```
Comment (nobu) [inline README.ja.md:155]:
```suggestion
[COPYING.ja](https://docs.ruby-lang.org/en/master/COPYING_ja.html) ファイルを参照してください．
```

**Comment by nobu** [comment]
Old files and cited LEGAL should be untouched, I think.

**Review by peterzhu2118 — CHANGES_REQUESTED** [review]
I don't think we should change release notes for old Rubies.

**Comment by BurdetteLamar** [comment]
Ok.  So we ignore `LEGAL` and `doc/NEWS/`.  Can anyone think of others, going forward?

**Comment by BurdetteLamar** [comment]
If no one objects, I'll close this and open a new PR without the offending changes.

**Comment by BurdetteLamar** [comment]
> If no one objects, I'll close this and open a new PR without the offending changes.

Doing so now.

**Pull request closed without merging**